.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: python
```

Comment:
```suggestion
dist: xenial
language: python
```

Reply:
Thanks, I would have sworn I included that. Not sure how I missed it!

Reply:
Actually it should be enabled by default. Probably it wasn't picket up due to some weird caching on the TravisCI side.